src/OrchardCore.Modules/OrchardCore.Security/Extensions/OrchardCoreBuilderExtensions.cs

```diff
@@ -0,0 +1,29 @@
+using Microsoft.Extensions.Configuration;
+using OrchardCore.Environment.Shell.Configuration;
+using OrchardCore.Security;
+
+namespace Microsoft.Extensions.DependencyInjection
+{
+    public static class OrchardCoreBuilderExtensions
+    {
+        public static OrchardCoreBuilder ConfigureSecuritySettings(this OrchardCoreBuilder builder, bool overrideAdminSettings = true)
+        {
+            builder.ConfigureServices((tenantServices, serviceProvider) =>
+            {
+                var shellConfiguration = serviceProvider.GetRequiredService<IShellConfiguration>().GetSection("SecuritySettings");
```

Comment:
@Piedone I'm not sure why I'm unable to get a proper settings from a configuration?!!

Reply:
They section is `"OrchardCore_SecuritySettings"`, not `"SecuritySettings"`.

Reply:
I though `OrchardCore_` prefix is added by `IShellConfiguration`, let me recheck with `OrchardCore_SecuritySettings`